modules/hitlnext/src/views/full/shared/components/AgentIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import { Colors, Icon } from '@blueprintjs/core'
+import React, { FC } from 'react'
+
+interface Props {
+  online: boolean
+}
+
+const AgentIcon: FC<Props> = props => {
+  function dotStyle() {
+    return {
+      top: -3,
+      right: -3,
+      position: 'absolute' as 'absolute',
+      width: 5,
+      height: 5,
+      backgroundColor: props.online ? Colors.GREEN1 : 'transparent',
+      borderRadius: 5
+    }
+  }
+
+  return (
+    <span style={{ position: 'relative' }}>
+      <Icon icon="headset"></Icon>
+      <span style={dotStyle()}></span>
```

Comment:
Use a class instead. You can use classnames to add a conditional class instead of building the style.
```js
import cx from 'classnames'
span className={cx(style.headsetIcon, { [style.online]: props.online })}>

// 
. headsetIcon {
background-color: transparent;
}
.online {
 background-color: green
}

Reply:
+1